xbmc/addons/AddonDatabase.cpp

```diff
@@ -732,7 +732,7 @@ bool CAddonDatabase::GetRepositoryContent(const std::string& id, VECADDONS& addo
       builder.SetVersion(version);
       builder.SetName(m_pDS->fv(4).get_asString());
       builder.SetSummary(m_pDS->fv(5).get_asString());
-      builder.SetDescription(m_pDS->fv(6).get_asString());
+      builder.SetDescription(m_pDS->fv("description").get_asString());
```

Comment:
if we use named columns here, should we do the same for all the other ones?

Reply:
Yea. The alternative is to explicitly list the fields in the SELECT statement above so that the field order matches the ints. I defer this decision to someone else.

Reply:
I would prefer to name the columns on the getters, so we can leave `SELECT *`.

Reply:
Then should I update the other ints to use column names?

Reply:
Would be nice if you had the time